src/NuGet/Microsoft.CodeAnalysis.Workspaces.Common.nuspec

```diff
@@ -22,10 +22,16 @@
         <dependency id="System.ObjectModel" version="$SystemObjectModelVersion$" />
         <dependency id="System.Text.RegularExpressions" version="$SystemTextRegularExpressionsVersion$" />
         <dependency id="System.Threading.Tasks.Parallel" version="$SystemThreadingTasksParallelVersion$" />
+        <dependency id="SQLitePCLRaw.bundle_green" version="$SQLitePCLRawbundle_greenVersion$" />
```

Comment:
@heejaechang Does this code actually run anywhere Workspaces runs, or is it still Windows only?

Reply:
Search in the project, it is not used. Removed from both the nuspec and the csproj

Reply:
@jasonmalinowski @heejaechang  Actually SQLitePCL is still used in the project. see Workspaces\Core\Desktop\Workspace\SQLite\Interop\SqlConnection.cs

Reply:
@JieCarolHu That file is Workspaces.Desktop, and isn't shipping in this package.

Reply:
Ah: this should go under the dependencies group then for net46 only, because that file only ships in the Desktop binary.

Reply:
deleted